.goreleaser.yml

```diff
@@ -15,7 +15,7 @@ builds:
   flags:
     - -trimpath
   ldflags:
-    - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
+    - '-s -w -X github.com/trocco-io/terraform-provider-trocco/version.ProviderVersion={{.Version}} -X main.commit={{.Commit}}'
```

Comment:
https://github.com/hashicorp/terraform-provider-aws/blob/f75b604ed2ccc1618be162393b623de3f232bb57/.goreleaser.yml#L41

I used this place as a reference.